islandora_solr.module

```diff
@@ -641,12 +641,12 @@ function islandora_solr_preprocess_islandora_objects_subset(&$variables) {
  */
 function islandora_solr_technical_difficulties(Exception $e) {
   // Only report the error once per screen.
-  $islandora_solr_error_reported = &drupal_static(__FUNCTION__, false);
+  $islandora_solr_error_reported = &drupal_static(__FUNCTION__, FALSE);
 
-  if ($islandora_solr_error_reported !== true) {
+  if ($islandora_solr_error_reported !== TRUE) {
     $message = "We are experiencing technical difficulties at this time and are working to fix the problem. Please try back later.";
```

Comment:
Final request. Could we have a way of customizing this message? Since it's front facing.. i know i know we did not have it before so why now? Because people i know will ask for it for sure.. maybe a hook-ie? i mean, its not a blocker... since you can always disable reporting at all...but could be cool? sorry